docs/src/tutorials/standalone/Soil/soil_energy_hydrology.jl

```diff
@@ -111,7 +111,7 @@ earth_param_set = LP.LandParameters(FT);
 ν_ss_gravel = FT(0.0);
```

Comment:
I noticed that the richards equation tutorial does not link correctly (at the very beginning of the file)